RIPS/rip-7560.md

```diff
@@ -69,6 +69,7 @@ smart contracts. This, however, is impossible without an addition of Native Acco
 | VERSION               | 1                                                                               |
 | MAGIC_VALUE_SENDER    | 0xbf45c166  // bytes4(keccak256("validateTransaction(uint256,bytes32,bytes)"))          |
 | MAGIC_VALUE_PAYMASTER | 0xe0e6183a  // bytes4(keccak256("validatePaymasterTransaction(uint256,bytes32,bytes)")) |
+| MAGIC_VALUE_SIGFAIL   | 0x31665494  // bytes4(keccak256("signature-error))                              |
```

Comment:
we removed the magics. probably needs rebase (or maybe the doc is stale with implementation..)

Reply:
Fixed the merge conflicts.